packit_service/service/templates/build_logs.html

```diff
@@ -0,0 +1,21 @@
+{% extends "layout.html" %}
+<!-- prettier-ignore -->
```

Comment:
What is the problem with `prettier` here?

Wouldn't be easier to ignore the whole file instead?

Reply:
Prettier is missing [Jinja Template](https://github.com/prettier/prettier/issues/5754) support. These ignore lines disable Prettier until the next HTML-tag to keep the template in a readable form.

It's a pain, but I wouldn't disable Prettier for the whole file, as there is still some value in uniform formatting for the parts which are not ignored.

Reply:
> These ignore lines disable Prettier until the next HTML-tag to keep the template in a readable form.

I know and that was my point if it makes sense to use it for templates.

Reply:
> I know and that was my point if it makes sense to use it for templates.

If you knew, why did you ask what the problem is? :confused: 

As I said above, I still see value in keeping Prettier enabled by default, and disable it only when it would otherwise make things very ugly.

Reply:
> If you knew, why did you ask what the problem is? confused

Sorry, I haven't read your comment properly and my comment does not make sense also to me... My bad.

> As I said above, I still see value in keeping Prettier enabled by default, and disable it only when it would otherwise make things very ugly.

:+1: That's what I was interested in. Thanks for the explanation.